Sources/Client/Client/ClientLogger.swift

```diff
@@ -227,6 +229,32 @@ public final class ClientLogger {
         lastTime = startTime
     }
     
+    public func logTaskDuration(_ description: String, task: () -> Void) {
```

Comment:
This api seems to be dangerous, because you can do `logger?.logTaskDuration( task )` because the task won't be executed if the logger is nil. I'd suggest removing the function completely.